src/DGmethods/DGmodel.jl

```diff
@@ -242,6 +242,54 @@ function nodal_update_aux!(f!, dg::DGModel, m::BalanceLaw, Q::MPIStateArray,
           knl_nodal_update_aux!(m, Val(dim), Val(polyorder), f!,
                           Q.data, dg.auxstate.data, dg.diffstate.data, t,
                           topology.realelems))
+
+end
+
+"""
+    cfl(localcfl::Function, dg::DGModel, m::BalanceLaw, Q::MPIStateArray,
```

Comment:
```suggestion
    cfl(local_cfl::Function, dg::DGModel, m::BalanceLaw, Q::MPIStateArray,
```
to improve readability